src/components/molecules/plan/plan.html

```diff
@@ -0,0 +1,48 @@
+<div class="plan_wrap">
```

Comment:
```suggestion
<div class="plan-wrap">
```
他のパーツはケバブケースで書いているので、統一するのが望ましいです！